src/tests/keywords/update-all

```diff
@@ -3,8 +3,6 @@
 #
```

Comment:
All of the `update-foo` tests should be left alone, as they're testing the `update` functionality.

Once `update` is gone, these tests can be deleted.